File: ADS 2023.2/AlgoritimosProgramacao/Algoritmos/While/lista_fabio/AlgoritimosProgramacao/Algoritmos/Questões/List-01/q01.js
import {question} from 'readline-sync'
function main(){
    //Input
    const sms = parseFloat(question("Enter a speed in m/s: "))

    //Processing
    const skm = sms * 3.6

    //Output
    console.log(`The entered speed of ${sms} m/s converted to k/h is: ${skm} k/h`)

}

main()

File: ADS 2023.2/AlgoritimosProgramacao/Algoritmos/While/lista_fabio/AlgoritimosProgramacao/Algoritmos/Questões/List-01/q01.py
def main():
#Input
    sms = int(input("Enter a speed in m/s: "))

#Processing
    skm = round(sms * 3.6)

#Output
    print(f"The entered speed of {sms} m/s converted to km/h is: {skm} k/h")
main()